recipes/find-kedro/meta.yml

```diff
@@ -0,0 +1,74 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "find-kedro" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  # If getting the source from GitHub, remove the line above,
+  # uncomment the line below, and modify as needed. Use releases if available:
+  # url: https://github.com/simplejson/simplejson/releases/download/{{ version }}/simplejson-{{ version }}.tar.gz
+  # and otherwise fall back to archive: 
+  # url: https://github.com/simplejson/simplejson/archive/{{ version }}.tar.gz
+  sha256: ddfb436bbb97b093e3aac862f3d67e5bcb015dac069ec1d5f84a315f9e573756
+  # sha256 is the preferred checksum -- you can get it for a file with:
+  #  `openssl sha256 <file name>`.
+  # You may need the openssl package, available on conda-forge:
+  #  `conda install openssl -c conda-forge``
+
+build:
```

Comment:
It looks like there is a python entry point for this library:

https://github.com/WaylonWalker/find-kedro/blob/master/setup.py#L31

so you need:

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points

Reply:
I added an entry points section